### PR TITLE
Feat/database/cr handler

### PR DESCRIPTION
**Pull request by WiktorSzymanski**
#89

### CODE REVIEW

**Review by BartoszDolata — APPROVED** [review]

File: backend/src/main/java/com/example/cms/ticket/TicketController.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import java.util.stream.Collectors;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/tickets")
+public class TicketController {
+    private final TicketService service;
+
+    @GetMapping
+    public ResponseEntity<List<TicketDto>> getUsers(
```
Comment (AdrianKokot) [inline backend/src/main/java/com/example/cms/ticket/TicketController.java:30]: I guess naming should be `getTickets`.

File: backend/src/main/java/com/example/cms/ticket/TicketController.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
+package com.example.cms.ticket;
```
Comment (AdrianKokot) [inline backend/src/main/java/com/example/cms/ticket/TicketController.java:1]: I think we will also need `getTicket` method for the "chat" view, but it's OK if it''s marked as separate task

File: backend/src/main/java/com/example/cms/ticket/TicketRoleSpecification.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    private Collection<Long> handlesPages;
+    private String email;
+
+    @Override
+    public Predicate toPredicate(Root<Ticket> root, CriteriaQuery<?> query, CriteriaBuilder criteriaBuilder) {
+        List<Predicate> predicates = new ArrayList<>();
+
+        Predicate conjunctionPred = criteriaBuilder.conjunction();
+
+        if (this.role == null) {
```
Comment (AdrianKokot) [inline backend/src/main/java/com/example/cms/ticket/TicketRoleSpecification.java:29]: To review later: I think that not logged in user should not be able to see tickets list, but we can confirm on our weekly call

**Review by AdrianKokot — APPROVED** [review]
Looks good!

File: backend/src/main/java/com/example/cms/ticket/TicketController.java (hunk 2 above)
Comment (WiktorSzymanski) [reply backend/src/main/java/com/example/cms/ticket/TicketController.java:1]: You can always use filters to get single ticket

**Pull request merged**